Common/Data/LeanDataWriter.cs

```diff
@@ -116,7 +116,7 @@ public void Write(IEnumerable<BaseData> source)
                             var fileData = currentFileData;
                             writeTasks.Enqueue(Task.Run(() =>
                             {
-                                WriteFile(file, fileData, data.Time);
```

Comment:
Yeah this should've been passing lastTime originally. But removing the date variable all together and pulling it from the data is fine 👍🏽